moda-contracts/src/ReleasesFactory.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
+import {IReleasesFactory} from "./interfaces/Releases/IReleasesFactory.sol";
+import {IReleases} from "./interfaces/Releases/IReleases.sol";
+import {IModaRegistry} from "./interfaces/IModaRegistry.sol";
+import {IReleasesRegistration} from "./interfaces/Releases/IReleasesRegistration.sol";
+
+contract ReleasesFactory is IReleasesFactory {
+    address public modaRegistry;
+    address public releasesMaster;
+    address public splitsFactory;
+
+    address[] public releases;
+
+    constructor(address modaRegistry_, address releasesMaster_, address splitsFactory_) {
+        modaRegistry = modaRegistry_;
+        releasesMaster = releasesMaster_;
+        splitsFactory = splitsFactory_;
+    }
+
+    function create(
+        address[] memory releaseAdmins,
+        string memory name,
+        string memory symbol,
+        address catalog
+    ) external {
+        address releasesClone = Clones.clone(releasesMaster);
+        releases.push(releasesClone);
```

Comment:
I don't think we need to store this here and store it in a Catalog. Prob just the Catalog

Reply:
Currently there is no way to fetch a Releases address from the Catalog, we can only pass in a releases address and check if it is registered or not, i'm doing this by keying the releases address to the releases owner 

`registeredReleasesContracts[releases] = releasesOwner` 

I realise this is a misleading name so i'm going to change that and also add the reverse so we can fetch the Releases address given the owner address and the owner address given the releases address.